File: python/minimum-moves-to-convert-string.py
class Solution:
    def minimumMoves(self, s: str) -> int:
        i = 0
        move_count = 0
        while i < len(s):
            if s[i] == "X":
                i += 3
                move_count += 1
            else:
                i += 1
        return move_count

File: rust/minimum-moves-to-convert-string.rs
impl Solution {
    pub fn minimum_moves(s: String) -> i32 {
        let mut i = 0;
        let mut move_count = 0;
        while i < s.len() {
            if s.chars().nth(i).unwrap() == 'X' {
                move_count += 1;
                i += 3;
            } else {
                i += 1;
            }
        }
        return move_count;
    }
}
